external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -139,8 +139,8 @@ public void onPartitionsRevoked(Collection<TopicPartition> partitions) {
 
         @Override
         public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
-            LOG.info("Partitions reassignment. [consumer-group={}, consumer={}, topic-partitions={}]",
-                    kafkaSpoutConfig.getConsumerGroupId(), kafkaConsumer, partitions);
+            LOG.info("Partitions reassignment. [task-ID={}, consumer-group={}, consumer={}, topic-partitions={}]",
```

Comment:
This is how I have derived the various styles : 
* taskId: [Task.java](https://github.com/apache/storm/blob/master/storm-client/src/jvm/org/apache/storm/daemon/Task.java#L127) has component ID named as `componentId`, so used `taskId` as the variable name for task ID. 
* Task-ID: This style is only being used in [the print statement](https://github.com/apache/storm/blob/master/external/storm-kafka/src/jvm/org/apache/storm/kafka/KafkaUtils.java#L290) and is consistent with the existing style.   
* task-ID: Only used once, consistent with the other variable names in the [log statement here](https://github.com/apache/storm/blob/master/external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java#L142). This can be renamed to `task-Id`. 
* I have used `taskID` as the variable name in rest of the files because `taskId` is the name of a function in same set of files.